samples/ambient-argo/README.md

```diff
@@ -0,0 +1,99 @@
+# Ambient Reference Architecture w/ Argo
+
+This repo contains a reference architecture for operating Istio Ambient Mesh with ArgoCD using GitOps.  It demonstrates best practices for leveraging Istio as part of an application platform.
+
+## :boom: DISCLAIMER
+
+Istio Ambient Mesh is still in Alpha, and is not suitable for production use.  Likewise, this reference architecture is of Alpha quality, and includes several rough edges, including:
+ * Cluster-Scoped upgrades cause known traffic loss, and have wide blast radius.
+ * The tag chart is forked from the primary istio repo, and needs to be merged and published
+ * CRDs are not currently upgraded
+
+## Getting Started
+
+This reference architecture assumes that you have an ArgoCD installation with:
+ * A [connected cluster](https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_cluster/) named `ambient-cluster`
+ * A connection to your repository (for private repos)
+
+To deploy Istio, supporting software, and the bookinfo sample application, copy this folder to the root of your repo and run:
+
+```bash
+read -p 'Please enter the URL to your repo:'
+OLD_REPO='{repo-placeholder}'
+find . \( -type d -name .git -prune \) -o -type f -name '*.yaml' -print0 | xargs -0 sed -i s,$OLD_REPO,$NEW_REPO,g
+argocd create application -f meta-application.json
+```
+
+## Repository Layout
+
+The meta-application.yaml file is an App-of-Apps that references all other applications needed for running Istio and the demo application via ArgoCD.  The diagram below demonstrates the deployment mechanism for each part of the platform.
+
+![architecture diagram][layout]
+
+## Principles
+
+The [GitOps Principles](https://opengitops.dev/) guide this reference architecture.  With the exception of [./meta-application.yaml](./meta-application.yaml) (the bootstrap file), Git is the source of truth for all components of our application.  Changes to all components of Istio, including the data plane, are initiated with a pull request.  Likewise, rollbacks are as simple as a revert.
+
+In particular, Istio Sidecars are known to violate the Declarative principle - the version of Istio injected in the sidecar is determined at runtime by the version of the injector at the time the pod was created.  Upgrading the injector does not cause the sidecar to upgrade, instead all injected pods must be restarted in order to upgrade their sidecars, which is an imperative operation, rather than a declarative one.
```

Comment:
How does this point about sidecar requiring pod restart change with ambient

Reply:
Ambient does not use sidecars, so manual restart is not necessary.  Eventually we will support running ambient with sidecars in the same mesh, for migration and for use cases that are not suitable for ambient, and then I imagine the sidecars will need the same special care that they need today.